docs/msbuild/how-use-project-sdk.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "How to: Reference an MSBuild project SDK | Microsoft Docs"
+ms.custom: ""
+ms.date: "11/04/2016"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology: 
+  - "vs-ide-sdk"
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+helpviewer_keywords: 
+  - "MSBuild, SDKs"
+  - "MSBuild, overriding DependsOn properties"
+  - "MSBuild, extending Visual Studio builds"
+  - "MSBuild, DependsOn properties"
+ms.assetid: b293fd37-59d7-4488-8027-edcad6b32965
+caps.latest.revision: 8
+author: "jeffkl"
+ms.author: "jeffkl"
+manager: angerlic
+ms.workload: 
+  - "multiple"
+---
+# How to: Use MSBuild Project SDKs
+In [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.md)] 15.0 a new concept was introduced called project software development kits (SDKs). 
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net46</TargetFramework>
+    </PropertyGroup>
+</Project>
+```  
+  
+During evaluation of the project, [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.md)] adds implicit imports at the top and bottom of your project:
+
+```xml
+<Project>
+    <!-- Implicit top import -->
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+    <PropertyGroup>
+        <TargetFramework>net46</TargetFramework>
+    </PropertyGroup>
+
+    <!-- Implicit bottom import -->
+    <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+</Project>  
+```  
+
+## Referencing a Project SDK
+ There are three ways to reference a project SDK
+
+1. Use the `Sdk` attribute on the `<Project/>` element:
+    ```xml
+    <Project Sdk="Microsoft.NET.Sdk">
+        ...
+    </Project>
+    ```
+    An implicit import is added to the top and bottom of the project as discussed above.  The format of the `Sdk` attribute is `Name[/Version]` where Version is optional.  For example, you can specify `Microsoft.NET.Sdk/1.2.3`.
+
+2. Use the `<Sdk/>` element for implicit imports:
+    ```xml
+    <Project>
+        <Sdk Name="Microsoft.NET.Sdk" Version="1.2.3" />
```

Comment:
I think we should not use `Microsoft.NET.Sdk` for examples with versions, since we expect to not want people to specify versions for that. Make up an example SDK?